GVFS/GVFS.Common/GitStatusCache.cs

```diff
@@ -311,20 +314,20 @@ private void RebuildStatusCacheIfNeeded(bool ignoreBackoff)
 
             if (needToRebuild)
             {
-                if (this.initialDelayTime > DateTime.MinValue)
-                {
-                    this.context.Tracer.RelatedInfo("GitStatusCache.RebuildStatusCacheIfNeeded: Generating new Status Cache... Status scan was delayed for: {0:0.##}s", (now - this.initialDelayTime).TotalSeconds);
-                }
-                else
-                {
-                    this.context.Tracer.RelatedInfo("GitStatusCache.RebuildStatusCacheIfNeeded: Generating new Status Cache...");
-                }
-
                 this.statistics.RecordBackgroundStatusScanRun();
 
                 bool rebuildStatusCacheSucceeded = this.TryRebuildStatusCache();
 
-                this.context.Tracer.RelatedInfo("GitStatusCache.RebuildStatusCacheIfNeeded: Done generating status. Cache is now: {0}", this.cacheState);
+                TimeSpan delayedTime = startTime - this.initialDelayTime;
+                TimeSpan statusRunTime = DateTime.UtcNow - startTime;
+
+                string message = string.Format("GitStatusCache.RebuildStatusCacheIfNeeded: Done generating status. Cache state: {0}. Status scan time: {1:0.##}s.", this.cacheState, statusRunTime.TotalSeconds);

```

Comment:
Use `$""` since that is what you are using below?

Reply:
Thanks - I went and used `String.Format` in both places to be consistent. I looked at using interpolation, but then the line was pretty long.